src/Directory.Packages.props

```diff
@@ -1,69 +1,74 @@
 <Project>
-    <PropertyGroup>
-        <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-        <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    </PropertyGroup>
-    <!-- NetFx project dependencies -->
-    <ItemGroup>
-        <PackageVersion Include="Microsoft.Data.SqlClient.SNI" Version="6.0.2" />
-        <PackageVersion Include="System.Buffers" Version="4.5.1" />
-        <PackageVersion Include="System.Memory" Version="4.5.5" />
```

Comment:
`System.ReadOnlyMemory` is referenced from this package in our public API in https://learn.microsoft.com/en-us/dotnet/api/microsoft.data.sqltypes.sqlvector-1.memory?view=sqlclient-dotnet-core-6.1#microsoft-data-sqltypes-sqlvector-1-memory

Is this expected to be loaded into runtime on its own in both .NET Framework v4.6.2 and .NET 8+?